### ISSUE TITLE
Bug Report: Remove Debug Warning broken

### ISSUE THREAD

**Issue by Ceiridge** [Ceiridge/Chrome-Developer-Mode-Extension-Warning-Patcher#72]
**Name the affected patch/problem (if available)**
Remove Debug Warning: Edge & Brave

**Comment by Jegadish003** [comment]
Hi! I'm having an issue with the installer, not sure if I'm making a mistake. I run the exe file and nothing happens. The "Do you want to allow this app to make changes" pop up shows up and I press Yes. The terminal flashes for a second before it vanishes and the GUI never opens. I have the .NET runtime installed.

**Comment by Ceiridge** [comment]
> Hi! I'm having an issue with the installer, not sure if I'm making a mistake. I run the exe file and nothing happens. The "Do you want to allow this app to make changes" pop up shows up and I press Yes. The terminal flashes for a second before it vanishes and the GUI never opens. I have the .NET runtime installed.

Do you have the .NET 6 desktop runtime installed? You can take a look in the event viewer (eventvwr) under Applications I think and look for an error related to this patcher.

**Comment by Jegadish003** [comment]
I have installed .NET 6.0.12 runtime and the event logs tell me the installation was successful. I checked the install location C:\Program files\dotnet and I can see the 6.0.12 and the dotnet.exe files all there. 
But running the exe file generates the same problem. Checking the event log for the failure tells me to check whether the .NET runtime is installed or needs to be updated. I am attaching the log text below. 


Log Name:      Application
Source:        .NET Runtime
Date:          18-12-2022 02:16:42 PM
Event ID:      1023
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      JEGA-LAP
Description:
Description: A .NET application failed.
Application: ChromeDevExtWarningPatcher.exe
Path: C:\Users\jegad\OneDrive\Desktop\Excess\Edge warning remove\ChromeDevExtWarningPatcher.exe
Message: You must install or update .NET to run this application.

App: C:\Users\jegad\OneDrive\Desktop\Excess\Edge warning remove\ChromeDevExtWarningPatcher.exe
Architecture: x64
Framework: 'Microsoft.WindowsDesktop.App', version '6.0.0' (x64)
.NET location: C:\Program Files\dotnet\

No frameworks were found.

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.WindowsDesktop.App&framework_version=6.0.0&arch=x64&rid=win10-x64


Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1023</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2022-12-18T06:16:42.3375731Z" />
    <EventRecordID>16845</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>JEGA-LAP</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Description: A .NET application failed.
Application: ChromeDevExtWarningPatcher.exe
Path: C:\Users\jegad\OneDrive\Desktop\Excess\Edge warning remove\ChromeDevExtWarningPatcher.exe
Message: You must install or update .NET to run this application.

App: C:\Users\jegad\OneDrive\Desktop\Excess\Edge warning remove\ChromeDevExtWarningPatcher.exe
Architecture: x64
Framework: 'Microsoft.WindowsDesktop.App', version '6.0.0' (x64)
.NET location: C:\Program Files\dotnet\

No frameworks were found.

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.WindowsDesktop.App&amp;framework_version=6.0.0&amp;arch=x64&amp;rid=win10-x64

</Data>
  </EventData>
</Event>

**Comment by Ceiridge** [comment]
> To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.WindowsDesktop.App&framework_version=6.0.0&arch=x64&rid=win10-x64

Have you tried doing what the error message has told you to do? If not, please try exactly this now.

**Comment by Jegadish003** [comment]
Yeah, the patcher works now. Thanks a lot for the Patcher and the help
